### PR TITLE
Post-release update for v1.3.8

### PR DESCRIPTION
**Pull request by Ashley-wenyizha**
**Is this a bug fix or adding new feature?**
Post-release update for v1.3.8
**What is this PR about? / Why do we need it?**

**What testing is done?** 
/on-hold

### CODE REVIEW

File: charts/aws-efs-csi-driver/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: aws-efs-csi-driver
 version: 2.2.5
```
Comment (wongma7) [inline charts/aws-efs-csi-driver/Chart.yaml:3]: 2.2.6

Comment (Ashley-wenyizha) [reply charts/aws-efs-csi-driver/Chart.yaml]: thanks! revised now

**Comment by wongma7** [comment]
/hold cancel

image is out 

```
aws ecr get-login-password | docker login --username AWS --password-stdin 602401143452.dkr.ecr.us-west-2.amazonaws.com
    docker manifest inspect 602401143452.dkr.ecr.us-west-2.amazonaws.com/eks/aws-efs-csi-driver:v1.3.8
```

**Pull request merged**